src/vpoller/worker.py

```diff
@@ -160,6 +162,40 @@ def load_config(self):
             'Worker Manager configuration: %s',
             self.config
         )
+
+
+    def check_workers(self):
+        for index in range(len(self.workers)):
```

Comment:
You can avoid using the `range(len(self.workers))` by simply iterating over the `self.workers` list by using `for worker in self.workers`.

Please update the above one, so that the code looks more Pythonic.